documentation/tutorial.md

```diff
@@ -68,3 +71,416 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploying Charmed MongoDB:
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
+```
+Every 2.0s: juju status --color                                                                                         ip-172-31-11-104: Fri Dec  2 11:24:30 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:24:30Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+```
+
+
+## Accessing MongoDB:
+*Disclaimer: this part of the tutorial accesses MongoDB via the `admin` user. It is not recommended that you directly interface with the admin user in a production environment. In a production environment [create a separate user](https://www.mongodb.com/docs/manual/tutorial/create-users/) and connect to MongoDB with that user instead.*
+
+For this part of the Tutorial we will access MongoDB - via the MongoDB shell `mongosh`. You can read more about the MongoDB shell [here](https://www.mongodb.com/docs/mongodb-shell/).
+
+
+### Install the MongoDB shell:
+While MongoDB is installed within the LXD containers that the application is hosted on, it is not installed on your machine. This means that you will have to install the MongoDB shell yourself. Install `mongosh` by following the official [instructions on how to install the MongoDB shell](https://www.mongodb.com/docs/mongodb-shell/install/#std-label-mdb-shell-install).
+
+
+### MongoDB URI:
+Connecting to the database requires a [URI](https://www.mongodb.com/docs/manual/reference/connection-string/). We use a URI of the format:
+```
+mongodb://<username>:<password>@<host>/<database name>?replicaSet=<replica set name>
+```
+
+Connecting via the URI requires that you know the values for `username`, `password`, `host`, `database name`, and the `replica set name`. We will now show you how to retrieve the necessary fields. 
+
+
+#### Retrieving the username:
```

Comment:
This is something that I'm noticing now but was present on previous PRs. I feel like this area has too many titles with little to explain under them. Instead of having a ton of `####` titles, maybe just streamline it with just a quick explanation of default values. I wouldn't say that explaining why those values are like that should be part of the tutorial.

Reply:
Agree about the titles. Do you think the explainations should be changed?